README.md

```diff
@@ -358,32 +399,44 @@ Any update of the KnowledgeStore requires running the script again to update the
-    winpty docker exec -v data:/data -it nlescstoryteller/query-builder-server //bin/bash
-    winpty docker exec -v data:/data -it nlescstoryteller/query-builder-daemon //bin/bash    
-    winpty docker exec -it nlescstoryteller/query-builder-client //bin/bash
-    winpty docker exec -it nlescstoryteller/uncertainty-visualization //bin/bash    
+winpty docker exec -v data:/data -it nlescstoryteller/storyteller //bin/bash
+winpty docker exec -v data:/data -it nlescstoryteller/query-builder-preprocessing //bin/bash
+winpty docker exec -v data:/data -it nlescstoryteller/query-builder-server //bin/bash
+winpty docker exec -v data:/data -it nlescstoryteller/query-builder-daemon //bin/bash    
+winpty docker exec -it nlescstoryteller/query-builder-client //bin/bash
+winpty docker exec -it nlescstoryteller/uncertainty-visualization //bin/bash    
 ```
 
 ## References
 
-P. Vossen, R. Agerri, I. Aldabe, A. Cybulska, M. van Erp, A. Fokkens, E. Laparra, A. Minard, A. P. Aprosio, G. Rigau, M. Rospocher, and R. Segers, “Newsreader: how semantic web helps natural language processing helps semantic web,” Special issue knowledge-based systems, elsevier, 2016. doi:http://dx.doi.org/10.1016/j.knosys.2016.07.013
-
-M. Rospocher, M. van Erp, P. Vossen, A. Fokkens, I. Aldabe, G. Rigau, A. Soroa, T. Ploeger, and T. Bogaard, “Building event-centric knowledge graphs from news,” Journal of web semantics, 2016.
-
-Zwaan van der J., M. van Meersbergen, A. Fokkens, S. ter Braake, I. Leemans, E. Kuijpers, P. Vossen, I. Maks, “Storyteller: visualizing perspectives in digital humanities projects,” in Proceedings of the 2nd ifip international workshop on computational history and data-driven humanities, Dublin, Ireland, Ireland, May 25, 2016.
-
-Van Hage, W.R., Malaisé, V., Segers, R., Hollink, L., Schreiber, G.: Design and use of the simple event model (sem). Web Semant. Sci. Serv. Agent. World Wide Web 9(2), 128–136 (2011)
+P. Vossen, R. Agerri, I. Aldabe, A. Cybulska, M. van Erp, A. Fokkens, E.
+Laparra, A. Minard, A. P. Aprosio, G. Rigau, M. Rospocher, and R. Segers,
+“Newsreader: how semantic web helps natural language processing helps semantic
+web,” Special issue knowledge-based systems, elsevier, 2016.
+doi:http://dx.doi.org/10.1016/j.knosys.2016.07.013
+
+M. Rospocher, M. van Erp, P. Vossen, A. Fokkens, I. Aldabe, G. Rigau, A. Soroa,
+T. Ploeger, and T. Bogaard, “Building event-centric knowledge graphs from news,”
+Journal of web semantics, 2016.
+
+Zwaan van der J., M. van Meersbergen, A. Fokkens, S. ter Braake, I. Leemans, E.
+Kuijpers, P. Vossen, I. Maks, “Storyteller: visualizing perspectives in digital
+humanities projects,” in Proceedings of the 2nd ifip international workshop on
+computational history and data-driven humanities, Dublin, Ireland, Ireland, May
+25, 2016.
+
+Van Hage, W.R., Malaisé, V., Segers, R., Hollink, L., Schreiber, G.: Design and
+use of the simple event model (sem). Web Semant. Sci. Serv. Agent. World Wide
+Web 9(2), 128–136 (2011) 
```

Comment:
The most important thing that I'm missing from these docs is this: so let's say I followed the instructions, and I now have two websites up and running, QueryBuilder and UncertaintyVisualization. Now what do I do?